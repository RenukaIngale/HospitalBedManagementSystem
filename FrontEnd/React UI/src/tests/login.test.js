import {shallow} from 'enzyme'
import Login from '../component/login'
import React from 'react'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import configureMockStore from 'redux-mock-store'
import {Provider} from 'react-redux'
Enzyme.configure({adapter:new Adapter()});
const mockStore=configureMockStore();
const store=mockStore({});
describe("Login Form Component",()=>{
    it("renders correctly",()=>{
        const wrapper=shallow(
        <Provider store={store}><Login/></Provider>
        );
        expect(wrapper).toMatchSnapshot();
    });
});