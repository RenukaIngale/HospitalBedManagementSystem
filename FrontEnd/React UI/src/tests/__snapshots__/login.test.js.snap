// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Form Component renders correctly 1`] = `ShallowWrapper {}`;
